cypress/utils/utils.ts

```diff
@@ -107,8 +107,8 @@ export function notExists(value: string): void {
 }
 
 export function selectFilter(filterName: string): void {
-    cy.wait(2000);
-    cy.get("div.pf-c-input-group").find(commonView.filterToggleButton).click();
+    cy.wait(4000);
+    cy.get("div.pf-c-input-group").eq(0).find(commonView.filterToggleButton).click({ force: true });
```

Comment:
Does this modification, work for all the earlier tests?

Reply:
yes